pkg/util/removeall/removeall.go

```diff
@@ -97,7 +97,7 @@ func RemoveAllOneFilesystem(mounter mount.Interface, path string) error {
 	fd.Close()
```

Comment:
nit: I would probably move `fd.Close` in a defer after opening the file and and assumning no errors were thrown.

Reply:
This looks like a deliberate decision in the original implementation. The comment right above that (line 96) says:
```
// Close directory, because windows won't remove opened directory.
```
if we were to defer instead, the fd would be closed after the remove() call, which could introduce issues on Windows systems.

Reply:
ah okay.